api/net/Areas/Subscriber/Controllers/ReportController.cs

```diff
@@ -316,9 +314,35 @@ public async Task<IActionResult> Generate(int id, [FromQuery] bool regenerate =
 
         var instances = _reportService.GetLatestInstances(id, user.Id);
         var currentInstance = instances.FirstOrDefault();
-        if (regenerate && currentInstance != null && currentInstance.SentOn.HasValue == false)
+        if (currentInstance == null)
```

Comment:
Fixed a bug where a report would be stuck after it was accepted by CHES but the SentOn date wasn't updated correctly.  I don't like this implementation.